command-parser.js

```diff
@@ -415,6 +415,7 @@ class CommandContext {
 				}
 				user.lastMessage = message;
 				user.lastMessageTime = Date.now();
+				user.talkedIn.push(room.id);
```

Comment:
Would rather not track this. Can you just scan the last, say, 20 lines of every chatroom or something?